Automattic/jetpack
# Add videopress product card to videopress recommendation step

#### Changes proposed in this Pull Request:

* This PR adds a product card to the VideoPress recommendation step if a site does not already have a VideoPress product or a plan that includes VideoPress.

#### Jetpack product discussion


pbNhbs-1iw-p2#comment-7724

#### Does this pull request change what data or activity we track or use?


This PR does not add any new analytics events, but can make use of the following existing event to record clicks on the product card CTA button, passing the step slug as a property of the event:
```
jetpack_recommendations_step_product_card_button_click
```

#### Testing instructions:






* Set up your test site with this PR, make sure the site has a Jetpack Free plan.
* Navigate to: `/wp-admin/admin.php?page=jetpack#/recommendations/videopress`
* You should see the product card for VideoPress showing on the right-hand side of the recommendation screen:

![Screen Shot 2022-04-19 at 10 25 20 AM](https://user-images.githubusercontent.com/18016357/164030711-21fa97e7-03bf-424f-845b-f6f546db0712.png)

* The CTA button on the product card should link to a checkout page with the `Jetpack VideoPress` product in the cart for the site that you are testing with.
* Repeat this test with a site that has a VideoPress product or a Complete plan - the product card should NOT show when viewing the VideoPress recommendation.
